crates/typst/src/math/equation.rs

```diff
@@ -78,18 +80,20 @@ pub struct EquationElem {
 
     /// The alignment of the equation numbering.
     ///
-    /// By default, the number is put at the `{end}` of the equation block. Both
-    /// `{start}` and `{end}` respects text direction; for absolute positioning,
-    /// use `{left}` or `{right}`.
+    /// By default, the alignment is `end+horizon`. For the horizontal
```

Comment:
```suggestion
    /// By default, the alignment is `end + horizon`. For the horizontal
```

Also, other alignments are written as `{end}`, probably this will generate links to alignment's documentation, so these should be wrapped too?

Reply:
thx! tbh I'm not sure what `{end} + {horizon}` will produce, because I don't know what pattern the doc generator is looking for.. but let's find out.